MapboxAndroidDemo/src/main/res/drawable/ic_get_app_24dp.xml

```diff
@@ -4,6 +4,6 @@
         android:viewportWidth="24.0"
         android:viewportHeight="24.0">
     <path
-        android:fillColor="@color/mapboxWhite"
```

Comment:
maybe this change and the ones below are not needed if you keep:
vectorDrawables.useSupportLibrary = true